apps/antalmanac/src/components/Calendar/CalendarCourseEventWrapper.tsx

```diff
@@ -22,8 +24,12 @@ export const CalendarCourseEventWrapper = ({ children, ...props }: CalendarCours
         (e: React.MouseEvent) => {
             e.preventDefault();
             e.stopPropagation();
-
-            setSelectedEvent(e, props.event);
+            if (props.event && (e.metaKey || e.ctrlKey)) {
```

Comment:
```suggestion

            if (props.event && (e.metaKey || e.ctrlKey)) {
```
nit: i like new lines 🙃